fastlane/lib/fastlane/plugins/templates/action.rb.erb

```diff
@@ -0,0 +1,35 @@
+module Fastlane
+  module Actions
+    class <%= plugin_name.capitalize %>Action < Action
+      def self.run(params)
+        UI.message("The <%= plugin_name %> plugin is working!")
+      end
+
+      def self.description
+        %q{<%= summary %>}
```

Comment:
I'm not a big fan of the %q, this is very confusing to the user, I'd much rather prefer using `"` and escaping the `"`. What do you think?